### DISCUSSION TITLE
Slack link is no longer active

### DISCUSSION THREAD

**Discussion by jens-egerod-ruter** [open-metadata/OpenMetadata#17906] [General]
I'm getting a "This link is no longer active. To join this workspace, you’ll need to ask the person who originally invited you for a new link." message when trying to join the slack workspace.

Is the website link broken?

**Comment by Prajwal214** [comment]
@jens-egerod-ruter You can Use this link [Slack](https://join.slack.com/t/openmetadata/shared_invite/zt-2r3k2hkg1-ToVJq~a6IkzBlRuUQUSJmA)

Can you share the screenshot of which page you faced this Issue

**Reply by jens-egerod-ruter** [reply]
From the slack button in the top right corner of the homepage

https://openmetadata.slack.com/join/shared_invite/zt-2owna83om-SyUnakOg8LW6rwMZqIILig#/shared-invite/error
![Screenshot 2024-09-20 at 11 40 30](https://github.com/user-attachments/assets/de82140d-a4d9-44da-9a30-672018b0bd52)
![Screenshot 2024-09-20 at 11 40 18](https://github.com/user-attachments/assets/24a07e9f-48b1-427c-80e3-b7f884dc5ce7)

**Reply by jens-egerod-ruter** [reply]
thank you, the link you provided works :)
